dev-tools/mage/integtest.go

```diff
@@ -192,6 +195,7 @@ func runInIntegTestEnv(mageTarget string, test func() error, passThroughEnvVars
 		// compose.EnsureUp needs to know the environment type.
 		"-e", "STACK_ENVIRONMENT=" + StackEnvironment,
 		"-e", "TESTING_ENVIRONMENT=" + StackEnvironment,
+		"-e", "GENERATE=" + Generate,
```

Comment:
This is a variable that specific to Filebeat so rather than putting it into the framework I think it should be passed in from the Filebeat magefile.go.

`RunIntegTest` accepts a vararg list of variables that should be passed through from the current environment into the container.

Reply:
Like this (untested 😄 ):

```
$ git diff
diff --git a/filebeat/magefile.go b/filebeat/magefile.go
index 864c3fac3..6425b955d 100644
--- a/filebeat/magefile.go
+++ b/filebeat/magefile.go
@@ -215,5 +215,5 @@ func PythonIntegTest(ctx context.Context) error {
                args := devtools.DefaultPythonTestIntegrationArgs()
                args.Env["MODULES_PATH"] = devtools.CWD("module")
                return devtools.PythonNoseTest(args)
-       })
+       }, "GENERATE")
 }
```

Reply:
I tested this ^ approach locally and it works.

Reply:
Thanks Andrew! I just pushed the changes